lib/vsc/install/shared_setup.py

```diff
@@ -501,6 +515,30 @@ def make_release_tree(self, base_dir, files):
             self.copy_file(os.path.join(REPO_BASE_DIR, fn), os.path.join(base_dir, fn))
 
 
+class vsc_sdist_rpm(vsc_sdist):
+    """Manipulate the shebang in all scripts"""
+
+    def make_release_tree(self, base_dir, files):
+        vsc_sdist.make_release_tree(self, base_dir, files)
+
+        if self.distribution.has_scripts():
+            # code based on sdist add_defaults
+            build_scripts = self.get_finalized_command('build_scripts')
+            scripts = build_scripts.get_source_files()
+
+            log.info("scripts to check for shebang %s" % (scripts))
+            pyshebang_reg = re.compile(r'\A'+SHEBANG_ENV_PYTHON)
+            for fn in scripts:
+                first_line = open(os.path.join(base_dir, fn)).readline()
+                if pyshebang_reg.search(first_line):
+                    log.info("going to adapt shebang for script %s" % fn)
+                    dest, code = self._recopy(base_dir, fn)
+                    code = pyshebang_reg.sub(SHEBANG_PYTHON_E, code)
```

Comment:
I'm trying to work out what this does in the face of code that's already been fixed (i.e. is already using `-E`). I think this doubles the `-E` which isn't ideal.
```
In [1]: testA = r'''#!/usr/bin/env python
   ...: print 'hello'
   ...: '''

In [2]: testB = r'''#!/usr/bin/env python -E
   ...: print 'hello'
   ...: '''

In [3]: import re

In [4]: p = re.compile(r'\A' + '#!/usr/bin/env python')

In [5]: p.search(testA)
Out[5]: <_sre.SRE_Match at 0x7f19ea8acc60>

In [6]: p.sub('#!/usr/bin/env python -E', testA)
Out[6]: "#!/usr/bin/env python -E\nprint 'hello'\n"

In [7]: p.sub('#!/usr/bin/env python -E', testB)
Out[7]: "#!/usr/bin/env python -E -E\nprint 'hello'\n"
```

Letting it search to the end of the line looks like it works:

```
In [20]: p = re.compile(r'\A' + '#!/usr/bin/env python.*')

In [21]: p.sub('#!/usr/bin/env python -E', testA)
Out[21]: "#!/usr/bin/env python -E\nprint 'hello'\n"

In [22]: p.sub('#!/usr/bin/env python -E', testB)
Out[22]: "#!/usr/bin/env python -E\nprint 'hello'\n"
```